Neos.Cache/Classes/Backend/PdoBackend.php

```diff
@@ -112,11 +113,12 @@ public function initializeObject()
      * @param string $entryIdentifier An identifier for this specific cache entry
      * @param string $data The data to be stored
      * @param array $tags Tags to associate with this cache entry
-     * @param integer $lifetime Lifetime of this cache entry in seconds. If NULL is specified, the default lifetime is used. "0" means unlimited liftime.
+     * @param integer $lifetime Lifetime of this cache entry in seconds. If NULL is specified, the default lifetime is used. "0" means unlimited lifetime.
      * @return void
      * @throws Exception if no cache frontend has been set.
      * @throws \InvalidArgumentException if the identifier is not valid
      * @throws InvalidDataException if $data is not a string
+     * @throws \Neos\Utility\Exception\FilesException
```

Comment:
Is this really correct? FilesException in a DB based backend?

Reply:
Yes, because it might create an SQLite DB, in the filesystem…

Reply:
Okay, can we somehow document that then? `@throws ... when using SQLite storage and the according file operation fails`